dirty_cat/tests/test_table_vectorizer.py

```diff
@@ -463,3 +464,60 @@ def test_check_name_change():
     """Test that using SuperVectorizer raises a deprecation warning"""
     with pytest.warns(FutureWarning):
         SuperVectorizer()
+
+
+def test_handle_unknown():
+    """
+    Test that new categories encountered in the test set
+    are handled correctly.
+    """
+    X = _get_clean_dataframe()
+    # Test with low cardinality and a StandardScaler for the numeric columns
+    sup_vec = SuperVectorizer(
```

Comment:
You should be using TableVectorizer, and not SuperVectorizer which is no deprecated.